workflow-core/src/main/java/com/squareup/workflow/Reactor.kt

```diff
@@ -179,19 +182,59 @@ interface Reactor<S : Any, E : Any, out O : Any> : WorkflowPool.Launcher<S, E, O
 }
 
 /**
- * Use this to implement [WorkflowPool.Launcher.launch].
+ * Use this only to create a top-level workflow. If you're implementing [Launcher.launch], use
+ * [doLaunch].
  */
-fun <S : Any, E : Any, O : Any> Reactor<S, E, O>.doLaunch(
+fun <S : Any, E : Any, O : Any> Reactor<S, E, O>.startRootWorkflow(
```

Comment:
This method is bullshit, will delete later (#44).

Reply:
#49